### PR TITLE
Add min/max to form collection

### PR DESCRIPTION
**Pull request by bjorvack**
Adds support for a min and max data attribute to limit/setup a form collection

### CODE REVIEW

File: src/js/Framework/FormCollection.js [hunk 1]
```diff
@@ -26,6 +29,11 @@ const FormCollection = function (element) {
       dragClass: 'collection-item-selected'
     })
   }
+
+  // Add minimum items
+  if (this.numberOfItems < this.minimumItems) {
```
Comment (jonasdekeukelaere) [inline src/js/Framework/FormCollection.js:34]: Does this not have to be a foreach? If minimum items is 3, only 1 item will be added?

File: src/js/Framework/FormCollection.js [hunk 2]
```diff
@@ -1,20 +1,23 @@
 import Sortable from 'sortablejs'
 
 const FormCollection = function (element) {
-  const addButton = element.querySelector('[data-role="collection-add-button"]')
+  this.addButton = element.querySelector('[data-role="collection-add-button"]')
```
Comment (tijsverkoyen) [inline src/js/Framework/FormCollection.js:4]: Is this best practice? In the `addItem`-method you also have the element available.

File: src/js/Framework/FormCollection.js [hunk 3]
```diff
@@ -51,6 +61,10 @@ const addItem = function (event, element) {
     removeItem(event)
   })
 
+  if (this.maximumItems && this.numberOfItems >= this.maximumItems) {
+    this.addButton.setAttribute('disabled', 'disabled')
+  }
+
   document.dispatchEvent(new Event('added.collection.item'))
```
Comment (tijsverkoyen) [inline src/js/Framework/FormCollection.js:68]: Maybe also check if this needs to be done. If they remove the disabled attribute from the button they still can add an item.

Comment (tijsverkoyen) [reply src/js/Framework/FormCollection.js:68]: Maybe even at the start of the method.

File: src/js/Framework/FormCollection.js (hunk 1 above)
Comment (bjorvack) [reply src/js/Framework/FormCollection.js]: fixed f6afd9f7b1a1c625450d35bea98d6daffcbff6d1

**Review by daphneslootmans — APPROVED** [review]

**Pull request merged**